### PR TITLE
feat: add catppuccin theme

### PR DESCRIPTION
**Pull request by ghost**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Adds the Catppuccin theme

### CODE REVIEW

File: themes/catppuccin.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "./schema.json",
+  "author": {
+    "name": "enno",
+    "twitter": "@YOUR TWITTER HANDLE",
```
Comment (fedeci) [inline themes/catppuccin.json]: Can you add your twitter username or remove this property, please?
```suggestion
    "twitter": "@YOUR TWITTER HANDLE",
```

**Review by fedeci — CHANGES_REQUESTED** [review]

File: themes/catppuccin.json (hunk 1 above)
Comment (ghost) [reply themes/catppuccin.json]: Fixed!

**Comment by ghost** [comment]
Fixed by removing the twitter field

**Review by fedeci — APPROVED** [review]
Thanks!

**Pull request merged**